main.py

```diff
@@ -5,18 +5,21 @@
 '''
 List of available commands:
 
-    read_sensor_temperature()
-    read_sensor_humidity()
-    read_sensor_gas()
-    read_sensor_distance()
-    read_sensor_motion()
+    print("*text*")
```

Comment:
Please remove the star sign since it could confuse people by letting them think that stars are always necessary to write strings.

Better:
`print("this is my message")`

Reply:
Changed "*text*" to your suggested "this is my message"